CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+## Contributing to PatternFly Elements
+
+[fork]: https://github.com/patternfly/patternfly-elements/fork
+[pr]: https://github.com/patternfly/patternfly-elements/compare
+[cc]: https://www.conventionalcommits.org/en/v1.0.0/#summary
+[doc]: https://patternfly.github.io/patternfly-elements/getting-started/
+
+
+Welcome Flyers!  We're so excited that you're considering contributing to PatternFly Elements, the web component implementation of the PatternFly design system.
+
+Contributions to this project are [released](https://help.github.com/articles/github-terms-of-service/#6-contributions-under-repository-license) to the public under the [project's open source license](LICENSE.md).
+
+Please note that this project is released with a [Contributor Covenant Code of Conduct](CODE_OF_CONDUCT.md). By participating in this project you agree to abide by its terms.
+
+## Submitting a pull request
+
+1. [Fork][fork] and clone the repository (see the Quick start in the [README][README.md])
+2. Create a new branch.  Please use the [conventional commit](cc) formatting for the branch name.
+    > `git checkout -b feat-my-branch-name`
+3. Make your change, [add tests](https://patternfly.github.io/patternfly-elements/develop/step-3/), and make sure the automated testing passes.
+4. Push to your fork and [submit a pull request][pr].
```

Comment:
should this list mention use of a PR template?

Reply:
Sure, do you have any ideas about how to word that?

Reply:
I took a stab at this